docs/products/kafka/howto/provectus-kafka-ui.rst

```diff
@@ -0,0 +1,88 @@
+Use Provectus® UI for Apache Kafka® with Aiven for Apache Kafka®
+================================================================
+
+`Provectus® UI for Apache Kafka® <https://github.com/provectus/kafka-ui>`_ is a popular Open-Source web GUI for Apache Kafka® management that allows you to monitor and manage Apache Kafka® clusters.
+
+Prerequisites
+-------------
+
+To connect Provectus® UI for Apache Kafka® to Aiven for Apache Kafka® you need to create a :doc:`Java keystore and truststore containing the service SSL certificates <keystore-truststore>`. 
+
+Furthermore, you need to collect the following information:
+
+* ``APACHE_KAFKA_HOST``: The Aiven for Apache Kafka hostname
+* ``APACHE_KAFKA_PORT``: The Aiven for Apache Kafka port
+* ``KEYSTORE_FILE_NAME``: The name of the Java keystore containing the Aiven for Apache Kafka SSL certificates
+* ``TRUSTSTORE_FILE_NAME``: The name of the Java truststore containing the Aiven for Apache Kafka SSL certificates
+* ``SSL_KEYSTORE_PASSWORD``: The password used to secure the Kava keystore
+* ``SSL_KEY_PASSWORD``: The password used to secure the Kava key
+* ``SSL_TRUSTSTORE_LOCATION``: The password used to secure the Kava truststore
+* ``SSL_TRUSTSTORE_PASSWORD``: The password used to secure the Kava truststore
+* ``SSL_STORE_FOLDER``: The absolute path of the folder containing both the truststore and keystore
+
+
+Run Provectus® UI for Apache Kafka® on Docker or Podman
+-------------------------------------------------------
+
+
+Share keystores with non-root user
+''''''''''''''''''''''''''''''''''
+
+Since container for Provectus® UI for Apache Kafka® uses non-root user, to avoid permission problems, while keeping the secrets safe, perform the following steps (see example commands below):
+
+1. Create separate directory for secrets::
+
+    mkdir kafka-secrets
+
+
+2. Restrict the directory to current user::
+
+    chmod 700 kafka-secrets
+
+3. Copy secrets there (replace the ``KEYSTORE_FILE_NAME`` and ``TRUSTSTORE_FILE_NAME`` with the keystores and truststores file names)::
+
+    cp KEYSTORE_FILE_NAMETRUSTSTORE_FILE_NAME kafka-secrets
+
+4. Give read permissions for secret files for everyone::
+
+    chmod +r kafka-secrets/*
```

Comment:
There is no need. Or, there should be probably an explanation, why it's safe, since you asked about it. On the step (2) we make the `kafka-secrets` readable only by the current user, on step (4) we indeed make secrets world-readable, but since they are in the user-readable directory, it should be safe (unless you copy the secrets out of this directory).

Proof:
```
$ sudo -Hi
# sudo -u nobody -i
$ cd /home/rominf/Downloads/
$ ls kafka-secrets/
ls: cannot open directory 'kafka-secrets': Permission denied
```

@aiven/team-security Am I correct? Or, is there a better way for making user-readable secrets available for non-root user inside Docker/Podman?

Reply:
this looks good for the purpose (not readable outside the container, but user inside can read the file because it is directly mounted). Slight optimisation can be to use `ro` in the bind argument in the `docker run` command,

```
docker run -p 8080:8080 \
        -v SSL_STORE_FOLDER/TRUSTSTORE_FILE_NAME:/client.truststore.jks:ro \
        -v SSL_STORE_FOLDER/KEYSTORE_FILE_NAME:/client.keystore.p12:ro \
```

Reply:
Thanks @staaldraad !